### PR TITLE
Make Newsletter Parser

### PR DESCRIPTION
**Pull request by shravan1k**
### Administrative Info
Monday Board ID: 940408300
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Made a parser that reads from a txt file and generates the newsletter
- Create logo as an image option
- A few CSS changes

### Testing
How did you confirm your changes worked? 
- Ran the diff command on the html we wrote and the html that was generated to find the differences. None were present.
- Viewed in browser and sent test emails to see if the newsletter was displayed correctly.

The latest newsletter was sent out using this parser.

### CODE REVIEW

**Review by eliasfang — APPROVED** [review]
Yep, I remember you showed us how it worked. Super cool!

File: py/newsletter.py [hunk 1]
```diff
@@ -14,36 +14,38 @@
 # USER VARIABLES
 # greetings
 greet_title = 'HAPPY WEEK * EVERYONE'
+greet_sub = 'Here are some things to get excited about!'
```
Comment (wes-chen) [inline py/newsletter.py:17]: so these don't get taken out with the parser?

**Review by wes-chen — APPROVED** [review]
looks good, I'd like to see some thorough documentation on how to use the parser (so that the next VP Operations can use it)

File: py/newsletter.py (hunk 1 above)
Comment (shravan1k) [reply py/newsletter.py:17]: They can be changed through the txt file. Those are left there as defaults.

**Pull request merged**